src/basic/src/ScalarSequence.C

```diff
@@ -1729,14 +1729,14 @@ ScalarSequence<T>::subBasicHistogram(
   for (unsigned int j = 0; j < dataSize; ++j) {
     double value = m_seq[j];
     if (value < minHorizontalValue) {
-      bins[0] += value;
+      bins[0]++;
```

Comment:
OK, I see that those were different lines in the same file with copy-pasted code. Thanks for the clarification.